benchmarks/README.md

```diff
@@ -0,0 +1,23 @@
+# Torchdynamo Benchmarks
+
+## What We Benchmark
+TorchDynamo provides a benchmark harness that takes care of uniformly benchmarking different models.  It interleaves runs of eager and dynamo to avoid machine noise/variability issues, and reports results based on medians along with P-values. 
+
+The runner integrates with models from TorchBenchmark, HuggingFace and TIMM suites and covers both training and inference.
+
+Training benchmarks approximate training by running the model forward, using a .sum().backward() call in place of the native loss function and skips the optimizer entirely.
+
+Inference benchmarks and Training benchmarks measure correctness by comparing dynamo and eager model outputs given fixed inputs and seeds.
+
+## Runbook
+There are a lot of flags in the benchmark runner, and it can be confusing to know which settings to use or what machine to run it on.  In order to support apples-to-apples comparison, we have provided the following 'standard' settings.
+
+We run benchmarks on AWS machines using 8xNVidia A100 64GB cards and (TODO) CPU.  (TODO instance type and OS/cuda versions) 
+
+### Training
+* TorchInductor + Triton
+* NVFuser
+
+
+### Inference
```

Comment:
@jansel also here? Idk if it's important to maintain commands for every backend but whichever ones you think are popular to run

Reply:
![image](https://user-images.githubusercontent.com/13822661/179052760-e4a396b2-b112-49c0-bd39-1b5e19ab537e.png)

This is what I have right now locally.

Reply:
cool. hmm, it almost makes me wonder if we should be putting these strings of flags into the README directly,

or instead we should add a new subparser to torchbench.py where strings like `ts_nnc_traininng` map directly to a set of args, and it gets maintained in the code instead of the README.

Then in the readme we can just point users to run simple commands like `torchbench.py --training ts_nvfuser` or something?

Reply:
I think we can do some cleanup. But overall, reducing the total number of args to shorten the command line might be a difficult goal to achieve. We have things options like dtype (float32/float16/amp) and device (cuda/cpu). And such things are likely to change.